src/next_state/stack.rs

```diff
@@ -211,3 +211,113 @@ pub trait NextStateStackMutExtClone: NextStateStackMut + Clone {
 }
 
 impl<S: NextStateStackMut + Clone> NextStateStackMutExtClone for S {}
+
+/// An extension trait for [`Commands`] that provides methods for operating on states with
+/// [`NextStateStack`] as their `Next` type.
+pub trait NextStateStackCommandsExt {
+    /// Push a new base state index to the stack.
```

Comment:
```suggestion
    /// Queue a command to push a new base state index to the stack.
```
Same thing for the other trait methods.

Reply:
Good call, this is more in line with Bevy's style!